samples/bootloader/src/main.c

```diff
@@ -104,11 +101,15 @@ extern u32_t _vector_table_pointer;
 #define VTOR SCB->VTOR
 #endif
 
-static void boot_from(u32_t *address)
+static void boot_from(const struct fw_firmware_info *fw_info)
 {
-	printk("Attempting to boot from address 0x%x.\n\r", (u32_t)address);
+	u32_t *vector_table = (u32_t *)fw_info->firmware_address;
+
+	printk("Attempting to boot from address 0x%x.\n\r",
+		fw_info->firmware_address);
 
-	if (!verify_firmware((u32_t)address)) {
+	if (!verify_firmware(fw_info->firmware_address)) {
+		printk("Failed to validate!\n\r");
```

Comment:
Should we try to mark failures as <err> and <inf> etc like zephyr? Could we re-use some print macros maybe?
```suggestion
		printk("err: Failed to validate!\n\r");

Reply:
I can't find that this is a convention. Maybe "err:" is added automatically by the logging subsys, but we are using printk instead.